src/Discord-Bridge.md

```diff
@@ -0,0 +1,36 @@
+# Discord Bridge
+
+## Setup
+
+First, you need to register a Discord application for your bridge. Discord applications can be registered and managed in the [Discord Developer Portal](https://discord.com/developers/applications/).
+
+1. Click on the "New Application" button in the upper right corner.
```

Comment:
Global: I believe we agreed that things we clicked on would be enclosed in backticks right? e.g. `"New Application"` should be \`New Application\`

Reply:
Oh yeah good point. I just blindly copy-pasted from the ems page. Will fix